linode/manager
# M3-5672: Events and Polling for Databases

## Description

- Adds polling to update the React Query Cache for Databases
- Adds event messages for Database events

## How to test

- Spin up a Database and check the events drawer for a relevant message
- Spin up a Database and ensure it goes from `provisioning` to `active` without refreshing (your going to have to wait a long time for it to provision)
- Create a Database from the API while Cloud Manager is open. Ensure the database you create from the API appears in the Landing table without needed to refresh
